hw/xbox/mcpx_apu.c

```diff
@@ -403,25 +409,44 @@ static const MemoryRegionOps vp_ops = {
     .write = vp_write,
 };
 
-static void scratch_rw(hwaddr sge_base, unsigned int max_sge,
+static void scratch_rw(MCPXAPUState *d,
```

Comment:
We should probably rename this to `scatter_gather_rw`.

It hasn't been a "scratch" accessor for some time, and will *very* soon be used for FIFO and other VP access.